scylla-cdc-driver3/src/main/java/com/scylladb/cdc/cql/driver3/Driver3SchemaBuilder.java

```diff
@@ -92,49 +92,24 @@ private void generatePrimaryKeyColumns() {
 
     private ChangeSchema.ColumnDefinition translateColumnDefinition(ColumnDefinitions.Definition driverDefinition, int index) {
         String columnName = driverDefinition.getName();
+
         ChangeSchema.DataType dataType = translateColumnDataType(driverDefinition.getType());
-        ChangeSchema.DataType baseType = null;
+        TableMetadata baseTableMetadata = baseTableMetadata();
+
         ChangeSchema.ColumnType baseTableColumnType = ChangeSchema.ColumnType.REGULAR;
-        boolean baseIsNonfrozenList = false;
         if (baseTablePartitionKeyColumnNames.contains(columnName)) {
             baseTableColumnType = ChangeSchema.ColumnType.PARTITION_KEY;
         } else if (baseTableClusteringKeyColumnNames.contains(columnName)) {
             baseTableColumnType = ChangeSchema.ColumnType.CLUSTERING_KEY;
-        } else {
-            /* Calculate whether corresponding base column is a non-frozen list.
```

Comment:
This comment was relevant to `baseIsNonfrozenList`. Only some portions of it are still relevant, especially the parts related to race conditions.

The race condition problem is much bigger - I created an issue about it: https://github.com/scylladb/scylla-cdc-java/issues/45 (the last paragraph). 

When I will push a PR that addresses the schema change, this problem will be solved by it, because my (future) implementation will explicitly manage those schemas. But this is out-of-scope of this PR, so this comment is removed for now. In order for its essence to get lost, I made the issue: https://github.com/scylladb/scylla-cdc-java/issues/45